### ISSUE TITLE
not unlocked by update

### ISSUE THREAD

**Issue by TYR23** [Aircoookie/WLED#979]
I enter the password, uncheck, after rebooting, access to ota is blocked

**Comment by raoulteeuwen** [comment]
Can you please provide a little more info? Did you at one time block OTA updates and your are now trying to unblock them? If so: you enter the password and still OTA is blocked?

If you mean something else, please see "If I report a new Issue, what do i need to share?" at https://github.com/Aircoookie/WLED/wiki/FAQ#if-i-report-a-new-issue-what-do-i-need-to-share the info that could help, one of the questions being 'has it ever worked?'. What password do you enter? What do you uncheck?
